src/h5web/visualizations/heatmap/HeatmapToolbar.module.css

```diff
@@ -4,23 +4,6 @@
   font-size: 0.875rem;
 }
 
-.toggler {
-  padding: 0 0.75rem;
-  composes: btn-clean from global;
-  display: flex;
-  align-items: center;
-  justify-content: left;
-}
-
-.toggler:hover,
-.toggler[aria-checked='true'] {
-  background-color: var(--primary-dark-bg);
-}
-
-.togglerLabel {
-  margin-left: 0.5rem;
-}
-
```

Comment:
Forgot to remove this when I extracted the toggler: this CSS exists now in Toggler CSS module.